foundry.toml

```diff
@@ -7,4 +7,7 @@ optimizer_runs  = 200                                   # The number of optimize
 solc_version    = '0.8.26'                              # Override for the solc version
 evm_version     = "cancun"
 
+# Allow agora-governor to resolve its own imports
+allow_paths = ["lib/agora-governor/src"]
```

Comment:
Not sure if you were encountering an issue but this might make verifying the contracts very annoying as sometimes foundry doesn't recognize the files when using relative paths. We can add it if you were having trouble building the artifacts but it works on my machine without.